fedimint-ui/apps/gateway-ui/src/components/WithdrawTab.tsx

```diff
@@ -199,17 +203,19 @@ const ConfirmWithdrawModal = (
         <Modal onClose={onModalClickCallback} isOpen={open} isCentered>
           <ModalOverlay />
           <ModalContent>
-            <ModalHeader>Confirm Withdrawal</ModalHeader>
+            <ModalHeader>{t('withdraw_tab.confirm_withdraw')}</ModalHeader>
             <ModalCloseButton />
             <ModalBody>
               <VStack alignItems='flex-start' justifyContent='space-between'>
                 <Box>
-                  <Text>Amount:</Text>
-                  <Text>{txRequest.amount} sats</Text>
+                  <Text>{t('withdraw_tab.amount')}</Text>
+                  <Text>
+                    {txRequest.amount} {t('withdraw_tab.sats')}
```

Comment:
Not blocking but just pondering that some of these common terms should be broken out into a common module for reuse.

Reply:
Ok! any thoughts on what kind of criteria we should set for commonality?
I'm thinking we should stick to just the general things
- Bitcoin
- sats
- Address
- Amount
- Confirm